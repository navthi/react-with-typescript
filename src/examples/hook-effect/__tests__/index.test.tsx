import * as React from 'react';
import {createRenderer} from 'react-test-renderer/shallow';

import Counter from '../index';

describe('examples/hook-effect', () => {
  const shallowRenderer = createRenderer();

  it('renders count', () => {
    const result = shallowRenderer.render(<Counter />);
    expect(result).toMatchSnapshot();
  });
});
