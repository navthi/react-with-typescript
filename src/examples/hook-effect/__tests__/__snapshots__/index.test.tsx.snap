// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`examples/hook-effect renders count 1`] = `
<div>
  <h1>
    Count: 
    0
  </h1>
</div>
`;
